default/python/AI/freeorion_tools/interactive_shell.py

```diff
@@ -89,8 +89,8 @@ def shell(msg):
     old_stdout = sys.stdout
     old_stderr = sys.stderr
 
-    sys.stdout = six.StringIO()
```

Comment:
Please don't remove the compatibility code.  I will do it after migration.  

My idea about using six is to make this PR clean of python code changes.
So I expect to see in this PR only changes that block migration (I am looking here when make code compatible, not all places are highlighted by tools).